commercialschedules-domain/src/main/init.db/initialize-db.sql

```diff
@@ -3,8 +3,6 @@
 
 \set ON_ERROR_STOP true
 
-CREATE USER dcsa_db_owner WITH PASSWORD '9c072fe8-c59c-11ea-b8d1-7b6577e9f3f5';
-CREATE DATABASE dcsa_template OWNER dcsa_db_owner;
+CREATE DATABASE dcsa_commercialschedules;
```

Comment:
`dcsa_cs` to keep it consistent with how we have been naming so far.

Reply:
Done the naming changes